src/App.test.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import App from './App';
+
+describe('App', () => {
+  it('renders app', () => {
+    const { container } = render((<App />));
+    expect(container).toContainHTML('<div');
+  });
+
+  describe('when new task is input', () => {
```

Comment:
```suggestion
  context('when new task is input', () => {
```
다양한 테스트들을 보기 쉽게 표현하신 것 같네요~ 이 경우에는 `describe-context-it` 형식에 맞춰 작성하면 더 어울릴 것 같아요~ 참고로 `context` 안에는 `with, without, when`만 사용하도록 권장합니다.

```
describe('render', () => {
  context('tasks가 없을 떄', () => {
    // Given
    const tasks = [];

    it('빈 메세지를 표시합니다.', () => {
      // When
      const { container } = render(<SomeComponent />);

      // Then
      expect(container).toHaveTextContent('투두가 없습니다.');
    });
  });
```

위의 예시 코드를 보면, `render는 tasks가 없을 때 빈 메시지를 출력한다.`와 같이 이해하기 쉽게 읽힙니다.

반드시 항상 `describe-context-it`의 구조를 적용할 필요는 없습니다. 시나리오에 맞게 고민해보시고 사용하시면 좋을 것 같아요~